siphon/simplewebservice/wyoming.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
+# Distributed under the terms of the MIT License.
+# SPDX-License-Identifier: MIT
+"""Read upper air data from the Wyoming archives."""
+
+from ..http_util import HTTPEndPoint
+from io import BytesIO
+import numpy as np
+import pandas as pd
+from metpy.calc import get_wind_components
+
+
+def get_upper_air_data(time, site_id):
```

Comment:
I think:
1. This function should be a `staticmethod` on `WyomingUpperAir`
2. It should just wrap the process of building a query and calling `get_data`.